### PR TITLE
Andrew lc patch 2

### PR DESCRIPTION
**Pull request by Andrew-LC**

### CODE REVIEW

**Review by Vishrut19 — APPROVED** [review]
Looks Good! Thanks for contributing 🚀🚀🚀

File: data/Andrew-LC.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    {
+      "name": "Follow me on Linkedin",
+      "url": "https://www.linkedin.com/in/andrew-lamichhane-34a397219/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/Andrew-LC",
+      "icon": "FaGithub"
+    },
```
Comment (kumarsonsoff3) [inline data/Andrew-LC.json:19]: Please remove this trailing comma `,` here. 

```suggestion
    }
```

Comment (Andrew-LC) [reply data/Andrew-LC.json]: Done 👍

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉 

<p align="center"><a href="https://linkfree.io/docs/how-to-guides/editing"></a></p>

**Pull request merged**